presto-common/src/main/java/com/facebook/presto/common/type/TypeSignature.java

```diff
@@ -252,7 +260,7 @@ VarcharEnumMap getVarcharEnumMap()
         Set<Integer> indices = new HashSet<>();
         Matcher enumMatcher = ENUM_PREFIX.matcher(signature);
         while (enumMatcher.find()) {
-            indices.add(enumMatcher.start());
+            indices.add(enumMatcher.end());
```

Comment:
Maybe I don't have the full picture. Why is this changed from start to end?

Reply:
The enum signature format is changed from `typename(enum:type{enum_map})` to
`enumtype(typename{enum_map})`